README.md

```diff
@@ -1 +1,101 @@
-# universal_notifications
\ No newline at end of file
+# universal_notifications
+
+[![build-status-image]][travis]
+[![pypi-version]][pypi]
+[![coverage-status-image]][codecov]
+
+**High-level framework for notifications**
+
+This project is intended to provide a convenient way to send notifications using multiple notification backends (e.g., e-mail, SMS, push).
+
+---
+## Setting up
+To start using **universal_notifications** please add `universal_notifications` to `INSTALLED_APPS` in your Django project, and then migrate the app: `./manage.py migrate universal_notifications`. 
+
+## Basic usage
+* [WebSocket notifications](#websocket-notifications)
```

Comment:
Something to check: In past pypi doesn't allow relative links.
Please check on pypitest.